pkg/object/objmetadata_test.go

```diff
@@ -90,6 +91,28 @@ func TestCreateObjMetadata(t *testing.T) {
 			expected: "",
 			isError:  true,
 		},
+		// Colon is allowed in the name for RBAC resources.
```

Comment:
Use named subtests instead of adding comments.

Reply:
Fixed.